Extension/walkthrough/build-project-linux.md

```diff
@@ -0,0 +1,13 @@
+# Build your C++ file with GCC on Linux
+
+1. Open the C++ source file that you want to build. Make sure this file is active (currently displayed) in the editor.
```

Comment:
"Currently displayed" doesn't mean active, i.e. there can be multiple files displayed but only one is active.